app/src/main/res/xml/pref_advanced_settings.xml

```diff
@@ -70,6 +70,12 @@
                 android:summary="@string/pref_summary_wear_logs_prefix"
                 android:textIsSelectable="true"
                 android:title="@string/pref_wear_logs_prefix" />
+            <CheckBoxPreference
```

Comment:
Okay, that makes sense.  Per my previous comment, I'd like to change the "Show Treatments" pref to "Sync Treatments" so that it is clearly distinct from the watch's "Show Treatments" as I don't think they should be unique.  Is it okay if I make changes in this PR still for this?